lerobot/common/utils/utils.py

```diff
@@ -158,6 +158,13 @@ def init_hydra_config(config_path: str, overrides: list[str] | None = None) -> D
         version_base="1.2",
     )
     cfg = hydra.compose(Path(config_path).stem, overrides)
+    if cfg.eval.n_episodes < cfg.eval.batch_size:
+        print("init_hydra_config: eval.batch_size ({}) > eval.n_episodes ({}), set eval.batch_size to {}").format(
+            cfg.eval.batch_size,
+            cfg.eval.n_episodes,
+            cfg.eval.n_episodes,
+        )
+        cfg.eval.batch_size = cfg.eval.n_episodes
```

Comment:
I am not sure to understand this code. Could we add a comment?

Reply:
Hi @Cadene,

As mentioned [here](https://github.com/huggingface/lerobot/pull/358#discussion_r1714645269), 
```
if cfg.eval.n_episodes < cfg.eval.batch_size:
    cfg.eval.batch_size = cfg.eval.n_episodes
```
is to avoid the program evaluates the policy for too many times, even if the user doesn't want to do so.

But you are right, I will add a comment here, instead of printing a message